openapi/definitions.yaml

```diff
@@ -1355,6 +1364,34 @@ ImpersonatedService:
     - service_id
     - user_groups
     - user_email
+ThirdPartyMessage:
+  type: object
+  properties:
+    attachments: 
+      type: array
+      items: 
+        $ref: "#/ThirdPartyAttachment"
+  required:
+    - attachments
+ThirdPartyAttachment:
+  type: object
+  properties:
+      id:
+        type: string
+        minLength: 1
+      content_type:
+        type: string
+        minLength: 1
+      name:
+        type: string
+        minLength: 1
+      url:
+        type: string
+        minLength: 1
+  required:
+    - id
+    - name
```

Comment:
I'm not sure if `name`is required.

Reply:
Fixed in [620fdc9](https://github.com/pagopa/io-functions-commons/pull/275/commits/620fdc9165792fdc6fc3644b2b4c3bd13d763879)